pkg/api/types.go

```diff
@@ -335,6 +335,8 @@ type Integration struct {
 	// IncludeBuiltImages determines if the release we assemble will include
 	// images built during the test itself.
 	IncludeBuiltImages bool `json:"include_built_images,omitempty"`
+	// ReferencePolicy is the policy to use when resolving references (Local or Source)
+	ReferencePolicy string `json:"reference_policy,omitempty"`
```

Comment:
It may be more convenient to change this to:
```golang
import imagev1 "imagev1 "github.com/openshift/api/image/v1"
...
ReferencePolicy *imagev1.TagReferencePolicyType `json:"reference_policy,omitempty"`
```
In this way we get validation for free as this type is already constrained to the values `Local` and `Source` and, when it's `nil`, when can default it.